lib/cdo/rack/upgrade_insecure_requests.rb

```diff
@@ -48,10 +48,10 @@ def call(env)
             "child-src blob: ",
             "script-src 'self' https: 'unsafe-inline' https://vaas.acapela-group.com 'unsafe-eval'",
             "style-src 'self' https: 'unsafe-inline'",
-            "img-src 'self' https: data: blob:",
+            "img-src 'self' https: data: blob: http://*.dev-code.org",
```

Comment:
should we instead use the full url for more safety? That's what @sureshc did for enabling the dev url for javabuilder

Reply:
Sure, yep I can do that